### ISSUE TITLE
.desc or .describe?

### ISSUE THREAD

**Issue by rauschma** [yargs/yargs#917]
I’m seeing both property names here: https://github.com/yargs/yargs/blob/master/docs/advanced.md

**Comment by bcoe** [comment]
@rauschma we accept `['describe', 'description', 'desc']` for the description of a command; do you think we should pick one and be consistent in the docs?

**Comment by florianb** [comment]
I'd like to vote for making this consistent.

**Comment by bcoe** [comment]
I agree that we should update all the docs to use `describe` or `description`, I'm open to either.
